test/test_tasks.rb

```diff
@@ -75,6 +77,39 @@ def with_stub_task_context(task_name)
                 end
             end
         end
+        
+        describe "GET /:namespace/:properties" do
+            it "returns property names" do
+                with_stub_task_context "task" do |task|
+                    get "/tasks/localhost/task/properties"
+                    assert_equal 201, last_response.status
+                    model = task.model.properties.to_h
+                    properties = Array.new
+                    expected = Hash[properties: properties]
+                    assert_equal expected, MultiJson.load(last_response.body, symbolize_keys: true)
+                end
+            end
+            
```

Comment:
Oh, read was available before, expected that is was already there, just added it